org.sonarlint.eclipse.ui/src/org/sonarlint/eclipse/ui/internal/markers/ShowIssueFlowsMarkerResolver.java

```diff
@@ -125,7 +125,7 @@ public static void showAnnotations(IMarker marker, ITextEditor textEditor) {
     try {
       positions = doc.getPositions(MarkerUtils.SONARLINT_EXTRA_POSITIONS_CATEGORY);
     } catch (BadPositionCategoryException e) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Either log or rethrow this exception. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1166)